crates/shim-sgx/src/lib.rs

```diff
@@ -38,7 +38,7 @@ pub const CSSA_1_PLUS_STACK_SIZE: usize =
     0x800000 - Page::SIZE - NUM_SSA * core::mem::size_of::<StateSaveArea>(); // 8MB - TCS - SSA
 
 /// FIXME: doc
-pub const ENCL_SIZE_BITS: u8 = 32;
+pub const ENCL_SIZE_BITS: u8 = 34;
```

Comment:
While you're here, can you address the FIXME comments above and below this?

Reply:
done